File: C Language/Week1/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height; //declare int variable Height
    do
    {
        height = get_int("Enter Size: ");
    }
    while (height < 1 || height > 8);

    for (int r = 0; r < height; r++)
    {
        for (int spaces = height - r - 1; spaces > 0; spaces--)
        {
            printf(" ");
        }
        for (int hash = 0; hash < r + 1; hash++) //to print hashes(#)
        {
            printf("#");
        }
        printf("  ");
        for (int r_hash = 0; r_hash < r + 1; r_hash++ )
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Python/Week6/pset6/mario/mario.py
from cs50 import get_int
incorrect_input = True

while incorrect_input:
    h  = get_int("Height: ")
    if ((h >= 1) and (h <=8)):
        incorrect_input = False
    else:
        continue

for i in range(1, h+1):
    print(f"{' ' * (h - i)}{'#' * i}  {'#' * i}")